packages/components/typography/src/text.tsx

```diff
@@ -0,0 +1,24 @@
+import styled from "styled-components";
+import { Element, ElementProps } from "@buoysoftware/anchor-layout";
+
+import {
+  textTransform,
+  TextTransformProps,
+} from "./custom_style_props/text_transform";
+
+export interface LegacyHtmlProps {
+  color?: string;
+  height?: string;
+  width?: string;
+}
+
+export type TextProps = ElementProps & TextTransformProps & LegacyHtmlProps;
+
+export const Text = styled(Element)<TextProps>`
+  font-family: -apple-system, BlinkMacSystemFont, San Francisco, Segoe UI,
+    Roboto, Helvetica Neue, sans-serif;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
```

Comment:
WDUT about creating a GlobalStyles component that handles resetting and using these base font families? I'm thinking doing so will be slightly more efficient as a class housing the font family information won't be created and added to every component at render.

On the other hand, it would mean that the text components aren't completely usable without the global styles also being included. (I can go either way)

Reply:
What about making a TextResetCSS that has just these styles so you can keep it in the typography package?

Reply:
I think for now I'm going to leave this as is, historically our font family was set on element and it's been working just fine. If we see performance issues at the CSS level we can investigate.